CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+6. Submit the pull request in the new form that appears
+
+### Where can I find the issue trackers?
+
+Go the repository for your interest group, and click the Issues tab (right under the title of the repository and to the
+right of the Code tab that is opened by default). By default this will show you all issues that are open, meaning they
+still have to be handled. There is an additional tab to show issues that have been dealt with and are closed. (you may
+still reopen and comment on them)
+
+### What counts as an issue?
+
+Any question, proposal, comment, response or whatever you want to raise can be considered an issue. Don't hold back!  We
+also encourage you to comment on any existing issues. Each issue forms a special thread for clearly delimited
+discussion.
+
+### How to raise an issue within an interest group?
+
+Post your issue to the **issue tracker** pertaining to your interest group, or to the
+[central](https://github.com/CLARIAH/IG/issues) group if it is more generic and spans multiple interest groups, or to
+the [use cases](https://github.com/CLARIAH/usecases/issues) issues if it specifically deals with use case.
+
+### How to add a use case?
+
+See [https://github.com/CLARIAH/usecases/blob/master/CONTRIBUTING.md](here) .
+
+### How to raise an issue about a use case?
+
+Open an issue on the [issue tracker for use cases](https://github.com/CLARIAH/usecases/issues)
+
+### Can we organize our documents in a DropBox/GoogleDrive/SURFdrive?
+
+No, please don't do that and keep them in the proper git repository instead to facilitate transparency. Otherwise we
+would have documents or copies of documents all over the place and we don't know where to look to find them or whether
+they are the proper latest version. Git is purposefully designed to solve this exact issue. The whole idea of using git
+and github for the interest groups is to consolidate version control and have a single public entry point.
+
+If you do have the document elsewhere, at least put a link to the document in the git repository (see next point).
+
+### We have a document on Google Docs/Office 365, can we link to it?
+
+Yes, you can link to the document from the git repository by adding simple markdown document to the git repository
+that contains a link to the document on the external platform. (be aware not to share a link with public write
+permissions in that case).
+
+Preferably, you can export the document and put it in the git repository itself. But you will have to repeat the
+procedure every time the document on the external platform changes. The version in git should always be authoritative.
+
+### Can't we just mail documents around to participants?
+
+No, this does not facilitate transparency, findability, reusability. Please use the repository and mail links to the
+document in the repository if needed.
+
+The notable exception here is when it concerns privacy-sensitive material, in that case the git repositories are **NOT**
+suitable.
```

Comment:
I honestly can't think of a practical reason to have privacy sensitive material in these repo's. Maybe we should add that, if you encounter something that you feel is unpublisheable, you should pose it as a question to the IG group? So we can come up with a proper way to handle this exceptional case?

Reply:
Indeed, I agree, that's the jest of the message. We could add a note as you suggest yes.